app.py

```diff
@@ -0,0 +1,75 @@
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from flask import Flask, render_template, request, redirect, url_for
+from pymongo import MongoClient
+
+import datetime
+import json
+import os
+import sys
+
+client = MongoClient(os.environ['MONGODB_URI'])
+db = os.environ['MONGODB_URI'].split('/')[-1]
+app = Flask(__name__, static_url_path='')
+
+
+def get_5_last_documents():
+    return list(client[db].coll.find().sort("ts", -1).limit(5))
+
+
+@app.route('/add_document', methods=['POST'])
+def add_document():
+    error_msg = None
+
+    try:
+        document = json.loads(request.form['document'])
+        document["ts"] = str(datetime.datetime.now())
+    except Exception as e:
+        error_msg = "Converting to JSON failed: {}".format(e)
+
+    if not error_msg:
+        try:
+            resp = client[db].coll.insert(document)
+        except Exception as e:
+            error_msg = "Inserting into MongoDB failed: {}".format(e)
+
+    results = get_5_last_documents()
+
+    return redirect(url_for(
+        'index',
+        msg=error_msg if error_msg else "Document successfully added",
+        notification=True,
+        success=False if error_msg else True
+    ))
+
+
+@app.route('/delete_doc', methods=['GET'])
+def delete_doc():
+    error_msg = None
+    oid = request.args.get('oid')
+
+    try:
+        client[db].coll.delete_one({"_id": ObjectId(oid)})
+    except Exception as e:
+        error_msg = "Deletion of object failed: {}".format(e)
+
+    return redirect(url_for(
+        'index',
+        msg=error_msg if error_msg else "{} successfully deleted".format(oid),
+        notification=True,
+        success=False if error_msg else True
+    ))
+
+
+@app.route('/')
+def index():
+    notification = True if request.args.get(
+        'notification') == 'True' else False
```

Comment:
Using `x if y else z` only really makes sense if it fits as a one-liner, it's more idiomatic to use a regular `if` statement otherwise.

However that's moot because this is more properly phrased as just:

``` python
notification = request.args.get('notification') == 'True'
```

(Feel free to wrap the RHS in parentheses if you prefer, but this is so short I don't think they'd help much.)